### DISCUSSION TITLE
JS internal packages: configuration guide

### DISCUSSION THREAD

**Discussion by lsg2099** [vercel/turborepo#4263] [Help] [answered]
Hello! I have imported 2 single React JS repos into a Turborepo and can start them as I was previously able to, happy days! However, my reasons for moving to Turborepo is that there is a very large React JS component that both of these repos/projects need to share (currently each repo has its own 'copy' of the component). I have tried to copy the example package "ui" when trying setting up this component as its own internal package, however I get the below error when starting the environment. Is there any documentation anywhere on how to setup a JS internal package rather than a TS internal package?
![image](https://user-images.githubusercontent.com/76950942/226338943-13708cd7-ba58-4756-9820-e5487b30f70b.png)

**Comment by anthonyshew** [comment, marked as answer]
This looks like an issue with Babel configuration. A quick search gave me this StackOverflow issue that looks similar? https://stackoverflow.com/questions/63005011/support-for-the-experimental-syntax-jsx-isnt-currently-enabled

While this is a legitimate error, I'm not sure that Turborepo plays a role here. If it does, we would appreciate a reproduction issue to get things solved. 🙏 Thank you!
